README.md

```diff
@@ -370,6 +370,40 @@ act -e pull-request.json
 
 Act will properly provide `github.head_ref` and `github.base_ref` to the action as expected.
 
+## Pass Inputs to Manually Triggered Workflows
+
+Example workflow file
+
+```yaml
+on:
+  workflow_dispatch:
+    inputs:
+      NAME:
+        description: "A random input name for the workflow"
+        type: string
+      SOME_VALUE:
+        description: "Some other input to pass"
+        type: string
+```
+
+Example JSON payload file conveniently named `payload.json`
+
+```json
+{
+  "action": "workflow_dispatch",
```

Comment:
The action isn't the event name. You will have to pass 'workflow_dispatch' as cli parameter, otherwise act will run a push event